lib/commands/execute.js

```diff
@@ -15,8 +15,11 @@ extensions.execute = async function (script, args) {
 extensions.executeMobile = async function (mobileCommand, opts = {}) {
   const mobileCommandsMapping = {
     shell: 'mobileShell',
+
     startLogsBroadcast: 'mobileStartLogsBroadcast',
     stopLogsBroadcast: 'mobileStopLogsBroadcast',
+
+    performEditorAction: 'mobilePerformEditorAction',
```

Comment:
Should we add https://github.com/appium/appium-uiautomator2-driver/blob/7af9866353a06f8a7f22675912082fe979811254/lib/commands/general.js#L81 as well?
Since no command error raised without `performEditorAction: 'mobilePerformEditorAction',` in `appium-uiautomator2-driver`

Reply:
yep, will add it later

Reply:
I want the other components to be published first